.github/workflows/terraform.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy publish app to PAAS
```

Comment:
Can we name the file after the name here, instead of `terraform`? Maybe `deploy-paas`?

Reply:
Renamed